client/app/scripts/liveblog-edit/views/main.html

```diff
@@ -92,11 +92,21 @@ <h3 class="page-nav-title">{{ blog.title }}</h3>
                     </div>
                 </div>
                 <!-- editor -->
-                <div class="panel--editor" ng-show="panelState === 'editor'">
-                    <header>
-                        Editor<button class="btn btn-info pull-right" ng-click="togglePreview()" translate>{{ preview? 'Edit': 'Preview' }}</button>
-                        <button class="btn pull-right" ng-click="askAndResetEditor()" translate>Reset</button>
-                    </header>
+                    <div class="panel--editor" ng-show="panelState === 'editor'">
+                        <div ng-if="!preview">
+                        <header>
+                            Editor<button class="btn btn-info pull-right" ng-click="togglePreview()" ng-disabled="actionStatus() && !isCurrentPostPublished()" translate>Preview
```

Comment:
why `ng-disabled="actionStatus() && !isCurrentPostPublished()"` regarding the preview button ?
Does it mean that we can't see the preview if the post hasn't been published ? Why ?